lib/Clownface.js

```diff
@@ -126,18 +208,39 @@ class Clownface {
     return Clownface.fromContext(context)
   }
 
+  /**
+   * Creates graph pointer to one or more blank node(s)
+   * @param {null|string|string[]|Clownface|Clownface[]} [values] blank node identifiers (uses generated if undefined of null)
+   * @returns {Clownface}
+   */
   blankNode (values) {
     return this.node(values, { type: 'BlankNode' })
   }
 
+  /**
+   * Creates graph pointer to one or more literal node(s)
+   * @param {string|string[]|boolean|boolean[]|number|number[]|Literal|Literal[]|Clownface|Clownface[]} values blank node identifiers (uses generated if undefined of null)
```

Comment:
This one is very similar (`undefined of null` where it should probably be `falsy`) as a previous one but a bit weirder (`values blank node identifiers`)